cubi_tk/isa_tpl/__init__.py

```diff
@@ -255,4 +258,21 @@ def run(args, parser, subparser):  # pragma: nocover
     if not args.tpl:  # pragma: nocover
         return run_nocmd(args, parser, subparser)
     else:
-        return args.isa_tpl_cmd(args, parser, subparser)
+        args.isa_tpl_cmd(args, parser, subparser)
```

Comment:
If running this would retrun an error (code) then it makes no sense to run the altamisa check, does it?